### ISSUE TITLE
Listing ng-options in the ThemisUI Docs

### ISSUE THREAD

**Issue by shaheershabbir** [clio/lib-themisui#203]
The ThemisUI website is really good and normally the only place I have to go but often times all of the various ng-options that can be used aren't listed for every component. Putting them on every component would a lot of repetition as well.

Is it possible to get another section called Component Options or something similar that just lists out what all of the various options are and maybe a quick example. The example section could just be something like "See in example in the th-input section."

**Comment by seanhealy** [comment]
A) Sorry for missing this for so long. But totally agree. I think we might also benefit from a general review of our component readmes to make sure they are fully up to date. Thoughts?

**Comment by shaheershabbir** [comment]
Yeah, in general they're really useful and I've just started looking around different

**Comment by shaheershabbir** [comment]
Accidentally closed it.

But with my experiences with the component library I've found the different ng-options to be the hardest part because some actually don't work with some components. Just having a list of the different options with a description and which components can and cant use that option would be good. Currently, I just do a trial and error approach.
